lib/util/prompt-suggestion.js

```diff
@@ -54,15 +54,21 @@ var getListDefault = function (question, defaultValue) {
  *
  * @param  {Object}       question Inquirer prompt item
  * @param  {String|Array} answer   The inquirer answer
+ * @param  {Boolean}      storeAll Should store default values
```

Comment:
I wonder if there's really any case where we don't want to store the default value. Given that we only update the default value of the question array, it wouldn't ever be a problem to store the default one right?